packages/mdc-button/README.md

```diff
@@ -40,38 +41,54 @@ npm install --save @material/button
 
 ## Usage
 
-### Flat
+### Button type
 
-```html
-<button class="mdc-button">
-  Flat button
-</button>
-```
+> Note: Examples and documents use generic `<button>`, but users can also apply
+`mdc-button` to `<a class="mdc-button">Link Button</a>` in cases where it is
+semantically correct.
 
-### Colored
+#### Text Button
 
 ```html
-<button class="mdc-button mdc-button--accent">
-  Colored button
+<button class="mdc-button">
+  Text button
 </button>
 ```
 
-### Raised
+#### Raised Button
 
 ```html
 <button class="mdc-button mdc-button--raised">
   Raised button
 </button>
 ```
 
-### Disabled
+### Button state
+
+#### Disabled
+
+Users can add `disabled` directly to the button element or set the fieldset containing
+the button to `disabled` to disable a button. Disabled buttons cannot be interacted with
+with and have no visual interaction effect.
 
 ```html
 <button class="mdc-button mdc-button--raised" disabled>
   Raised disabled button
 </button>
 ```
 
+### Colored
+
+Button have a default baseline color, but it could also be set to adopt the
```

Comment:
Suggestion: "MDC Buttons have a default baseline color, but it is also possible to adopt the"